plugin-core/src/main/java/appland/installGuide/InstallGuideEditor.java

```diff
@@ -242,8 +237,8 @@ private void addBaseProperties(@NotNull JsonObject json) {
 
         json.addProperty("page", type.getPageId());
         json.addProperty("userAuthenticated", settings.getApiKey() != null);
-        json.addProperty("analysisEnabled", settings.isAnalysisEnabled());
-        json.addProperty("findingsEnabled", settings.isEnableFindings());
+        json.addProperty("analysisEnabled", true);
```

Comment:
Rather than hard-coding this to `true`, can you just remove it?

Reply:
we could, but I wasn't sure what the webview would do with a missing property...

Reply:
Good point, I just checked and you can remove `findingsEnabled`, but don't remove `analysisEnabled`.